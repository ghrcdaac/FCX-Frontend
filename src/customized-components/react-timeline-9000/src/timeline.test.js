'use strict';
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from "enzyme-to-json"

import './setupTests';

import Timeline from 'timeline';

describe('<Timeline/>', function() {
  it('should have text', function() {
    const wrapper = shallow(
      <Timeline
        items={[]}
        groups={[]}
        startDate={{ date: "2017-05-17"}}
        endDate={{date: "2017-05-17"}}
        onInteraction={() => {}}
      />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
