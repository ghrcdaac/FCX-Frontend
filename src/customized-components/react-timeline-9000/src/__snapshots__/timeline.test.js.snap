// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timeline/> should have text 1`] = `
<div
  className="rct9k-timeline-div rct9k-id-r9k1"
>
  <AutoSizer
    className="rct9k-autosizer"
    disableHeight={false}
    disableWidth={false}
    onResize={[Function]}
    style={Object {}}
  >
    <Component />
  </AutoSizer>
</div>
`;
